Automattic/jetpack
# XMLRPC API: Introducing `jetpack_connection` flag.

The new flag `jetpack_connection` indicates that the site is connected to WordPress.com.
The existing flag `jetpack` assumes that Jetpack plugin itself is installed.

Since the endpoint itself is located in Jetpack anyway, both flags are hardcoded as `true`.
At the moment they are only needed to make the endpoint compatible with WP.com API.

The changes go along with for `D44063-code`.
The "Touched WP.com Files" diff is created manually: `D46835-code`.

#### Changes proposed in this Pull Request:
* Add new flag `jetpack_connection: true` to the `/sites/%s` API endpiont.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
1. Run unit tests: `yarn docker:phpunit --filter="WP_Test_Jetpack_Site_Json_Api_Endpoints::*"`
2. Open the URL (replace example.com with your domain name): `https://public-api.wordpress.com/rest/v1/sites/example.com/`, confirm that both flags are in there: `"jetpack":true, "jetpack_connection":true`

#### Proposed changelog entry for your changes:
No entry needed.
